xml/decoders/piko/PanePikoLichtDecoder_v5.1.xml

```diff
@@ -34,14 +34,36 @@
 			<revnumber>2</revnumber>
 			<date>2023-05-01</date>
 			<authorinitials>UG</authorinitials>
-			<revremark>Add SUSI decoder BR 93 TT</revremark>
+			<revremark>Add SUSI decoder BR 93 TT, add CV1018 for output test</revremark>
 		</revision>
 	</revhistory>
 	<name>SUSI LightDecoder</name>
 	<name xml:lang="de">SUSI LichtDecoder</name>
+	<row>
+		<label>
+			<text>
+				<!--Leerzeile-->
```

Comment:
I suggest changing these lines to:
`<!-- Leerzeile (empty line) -->`

That makes it much easier for non German speaking developers to understand what it's about.